packages/@webex/internal-plugin-voicea/package.json

```diff
@@ -41,8 +42,9 @@
     "sinon": "^9.2.4"
   },
   "scripts": {
-    "build": "yarn build:src",
-    "build:src": "webex-legacy-tools build -dest \"./dist\" -src \"./src\" -js -ts -maps",
+    "ls": "pwd",
```

Comment:
@Shreyas281299  why do we want to add the `ls` command here ?
Also `ls` is doing `pwd` ?  `pwd` is to get full path name of your current directory but `ls` is for listing files.

Reply:
@Neeraj-swarnkar  thanks. No this was added by mistake. I was testing something thats why I added this. Removing it now.